homeassistant/components/fitbit/sensor.py

```diff
@@ -185,9 +185,7 @@ def fitbit_configuration_callback(callback_data):
         else:
             setup_platform(hass, config, add_entities, discovery_info)
 
-    start_url = (
-        f"{get_url(hass, require_current_request=True)}{FITBIT_AUTH_CALLBACK_PATH}"
-    )
+    start_url = f"{get_url(hass, prefer_external=True)}{FITBIT_AUTH_CALLBACK_PATH}"
```

Comment:
We should not prefer external. Reasoning:

- We didn't before the change
- It makes this impossible to use for people with NAT loopback issues

The internal URL could be used by anybody.

```suggestion
    start_url = f"{get_url(hass)}{FITBIT_AUTH_CALLBACK_PATH}"
```